style.css

```diff
@@ -0,0 +1,92 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+main {
+  border-bottom-left-radius: 25px;
+}
+
+.portfolio-nav {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  height: 72px;
+}
+
+.portfolio-nav--logo {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+}
+
+.portfolio-nav--nav-options {
+  display: flex;
+  list-style: none;
+}
+
+.portfolio-nav--nav-options li {
+  padding: 0 20px;
+  font-weight: 500;
+  font-size: 15px;
+  color: #344563;
+}
+
+.portfolio-nav--nav-options a {
+  text-decoration: none;
+}
+
+.headline {
+  margin: 47px 20px 27px 20px;
+  background: url("images/Header\ bg@2x.png");
+  background-size: cover;
+  height: 721px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.headline--info {
+  margin: 0 237px;
+}
+
+.headline--info--name {
+  font-weight: 700;
+  font-size: 48px;
+  color: #091e42;
+  display: flex;
+  margin: 0;
+}
+
+.headline--info--desc {
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+  margin-top: 12px;
+}
+
+.headline--info h3 {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.headline--social-media {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  align-items: baseline;
+  width: 171px;
+  margin-top: 163px;
+  margin-left: 220px;
+  padding: 0;
+}
+
+.headline--social-media img {
```

Comment:
Please make sure that your design matches the Figma design because it's included in the requirements

Your Design:

![student2](https://user-images.githubusercontent.com/95412943/212046178-fb9c5a10-8a7a-40ec-a97a-292e0886a3c8.PNG)

The Figma design:

![figma2](https://user-images.githubusercontent.com/95412943/212046236-5f13bc86-df5c-477d-8d3c-fc37391212d3.PNG)

Reply:
Thanks for the review @Abdessittir, but isn't it for the next milestone to make it responsive?

Reply:
Hi @zdnahom, I hope you are doing great.
Yes, you can make it responsive afterward, but as the requirement said your mobile style should match the mobile version of the Figma design 😉.

If you have a question please let me know 👍